rxros_tf/include/rxros_tf/rxros_tf.h

```diff
@@ -77,15 +77,15 @@ namespace rxros
         auto send_transform() {
             return [=](auto&& source) {
                 tf::TransformBroadcaster transformBroadcaster;
-                source.observe_on(rxcpp::synchronize_new_thread()).subscribe(
+                source.subscribe_on(rxcpp::synchronize_new_thread()).subscribe(
                     [&](const tf::StampedTransform& stf) {transformBroadcaster.sendTransform(stf);});
                 return source;};}
 
 
         auto send_transform(const std::string &parent_frameId, const std::string &child_frameId) {
             return [=](auto&& source) {
                 tf::TransformBroadcaster transformBroadcaster;
-                source.observe_on(rxcpp::synchronize_new_thread()).subscribe(
+                source.subscribe_on(rxcpp::synchronize_new_thread()).subscribe(
```

Comment:
I suggest removing observe_on here as well (not just replacing it with subscribe_on).

Basically, when we remove them, everything seems to become synchronous, based on the input source (typically but not necessarily a topic subscription).  If the programmer wants to parallelize more they should insert observe_on (if they want to parallelize the pipeline) or subscribe_on (if they want to just parallelize the sending).  Also we should probably explain subscribe_on and observe_on in the documentation.